src/segments/language.go

```diff
@@ -246,6 +248,22 @@ func (l *language) inLanguageContext() bool {
 	return l.inContext()
 }
 
+func (l *language) buildDefaultVersionURL() {
```

Comment:
I don't see the added value here. It doesn't allow the user to add an extra URL, it uses `l.versionURLTemplate` which is baked in and produces the same output as `buildVersionURL` in that case.

Reply:
Not following, how do you use `l.versionURLTemplate` in `version_url_template` in case you want to extend that? I tried `{{ .versionURLTemplate }}` but that doesn't work. It can be seen in tests that the end result is 2 different URLs if the template is used.

Reply:
The functions are identical apart from `https://github.com/JanDeDobbeleer/oh-my-posh/blob/b7d752d87b8f7ab4cdc4ea3ad3e4b3382f10f0d2/src/segments/language.go#L268`. I don't see the use-case of having two release note URL's for the same version. Adjust towards what's needed in the config using `version_url_template`. 

Thus:

> open a URL with more information about the language version or release note

Where is this an actual use-case? Why don't you simply override the built-in one if you have a better one?